client/src/document/index.scss

```diff
@@ -161,6 +161,11 @@ a.new {
   */
   .index {
     columns: 300px;
+    margin-bottom: $base-spacing;
+
+    @media #{$mq-small-desktop-and-up} {
+      margin-bottom: $base-spacing * 2;
```

Comment:
I did play with setting this on line 164, too. I.e. the same on mobile.
Before:
<img width="656" alt="Screen Shot 2021-06-01 at 9 23 08 AM" src="https://user-images.githubusercontent.com/26739/120330807-2cba1e80-c2bb-11eb-8d54-5ff0d39bf33c.png">

After:
<img width="616" alt="Screen Shot 2021-06-01 at 9 24 07 AM" src="https://user-images.githubusercontent.com/26739/120330836-35125980-c2bb-11eb-8e9f-9aa15b1615fa.png">

Which, for the record I think is perfectly pretty and almost better. But it's not for me to say. I'm fine with more compact on mobile too.

Reply:
I feel like it is a tad much is perhaps too different than the amount of spacing we use for other things. Something to keep in mind though. Thanks, Peter.